packages/core/src/html/UlNodeProcessor.ts

```diff
@@ -0,0 +1,242 @@
+import { EmojiConvertor } from 'emoji-js';
+import isNumber from 'lodash/isNumber';
+import isString from 'lodash/isString';
+import { NodeOrText } from '../utils/node';
+
+interface IconAttributes {
+  icon?: string;
+  iconSize?: string;
+  className?: string;
+}
+
+type Size = {
+  fontSize: string;
+  imageSize: string;
+  squareSize: string;
+};
+
+function updateNodeRelations(iChild: NodeOrText, divChild: NodeOrText, children: NodeOrText[]): void {
+  iChild.next = divChild;
+  divChild.prev = iChild;
+
+  children.forEach((child) => {
+    child.parent = divChild;
+  });
+}
+
+function getStyleObject(node: NodeOrText): { [key: string]: string } {
+  let styleObject: { [key: string]: string } = {};
+  if (node.attribs?.style) {
+    const { style } = node.attribs;
+    styleObject = style.split(';').reduce((acc: Record<string, string>, rule: string) => {
+      const [property, value] = rule.split(':').map((str: string) => str.trim());
+      if (property) acc[property] = value;
+      return acc;
+    }, {});
+  }
+  return styleObject;
+}
+
+function createStyleString(styleObject: { [key: string]: string }): string {
+  return Object.entries(styleObject).map(
+    ([property, value]: [string, string]) => `${property}: ${value}`).join('; ');
+}
+
+function createIChild(
+  parent: NodeOrText,
+  icon: string,
+  size: Size,
+  className?: string,
+): NodeOrText {
+  const emoji = new EmojiConvertor();
```

Comment:
Is the feature set the same as the existing icon feature?

https://github.com/markdown-it/markdown-it-emoji/blob/master/lib/data/full.json

We have a small addition here as well: 
https://github.com/MarkBind/markbind/blob/master/packages/core/src/lib/markdown-it/patches/markdown-it-emoji-fixed.js

Reply:
Related: https://github.com/MarkBind/markbind/pull/2316#discussion_r1252081924

Reply:
Hmm, interesting list... just realise it...